modules/ROOT/pages/analytics-faq.adoc

```diff
@@ -30,10 +30,10 @@ If these messages are not present, check the application's log to see that you s
 "New message response notification... "
 ----
 
-In previous versions of Anypoint Data Gateway, an issue prevented RAML type proxies from working correctly. Ensure that you have the latest software where this issue has been fixed.
+In previous versions of Anypoint Data Gateway, an issue prevented RAML/OAS type proxies from working correctly. Ensure that you have the latest software where this issue has been fixed.
```

Comment:
@evangelina @fermujica why are we talking about Data Gateway here? Is it a typo?

Reply:
Yes, this is a type, @fermujica , can we change Anypoint Data Gateway for API Gateway?

Reply:
You are right. This was a typo: 6e591a5